Sources/Storage+Codable.swift

```diff
@@ -21,7 +21,7 @@ public extension Storage {
         })
     }
 
-    func load<T: Codable>(forKey key: String, as: T.Type) throws -> T {
+    func load<T: Codable>(forKey key: String, as: T.Type, maxAge: TimeInterval = .greatestFiniteMagnitude) throws -> T {
```

Comment:
This is great, I also want it to be like a enum Expiry with `never` and `date` option

Reply:
Very good idea 👍 20102cd86d4df3d9786a00e0982ef0c078b3e575

Reply:
just didn't use a date object. I prefer a `TimeInterval` for the following reason:
* request shouldn't be older as 5 minutes (TimeInterval: 300)

Reply:
Oh ja, I meant time interval too